### ISSUE TITLE
feature request - VR

### ISSUE THREAD

**Issue by py1sl** [fusion-energy/fusion_neutronics_workflow#37]
In this very nice looking workflow are there plans to add any automated variance reduction? Magic? You could imagine a check on the tally to determine if passing some metric and if not running some automated VR routines, depending on if global or target based VR is required.  Considering OpenMC VR is only just in most recent versions, I guess this might needs OpenMC developments as well.

**Comment by shimwell** [comment]
Yep Magic is being worked into OpenMC, I guess it is about 2 months away. I had a quick go at the weight windows but it is difficult without Magic.
